### ISSUE TITLE
Dark Mode & Speed Limit Feature!

### ISSUE THREAD

**Issue by imranext** [NidukaAkalanka/x-ui-english#179]
### Welcome

- [X] Yes, I'm using the latest major release. Only such installations are supported.
- [X] Yes, I'm using the supported system. Only such systems are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, log, etc).

### Description of the problem,screencshot would be good

Hello Admin,
Please add dark mode & speed limit feature for individual clients.

Thanks.

### Version of x-ui

<details>

```console
$ x-ui version
# Paste output here
```

</details>

### x-ui log or xray log

<details>

```console
# paste log here
```

</details>

**Comment by drno88** [comment]
yes I support. this is a much needed feature. or tell me how to use it using iptables for example...
